### DISCUSSION TITLE
OutポートendCogStateとlandingTarget

### DISCUSSION THREAD

**Discussion by Naoki-Hiraoka** [YutaKojio/hrpsys-base#49] [Q&A]
Outポート`endCogState`と`landingTarget`は現在使用されていますか? HrpsysSeqStateROSBridgeからROSに出力されているようですが、使用されている場合、現在動くサンプルなどはありますか?

**Comment by YutaKojio** [comment]
endCogStateは壁に手をついて転倒回避するときに使う値で，たぶんD論式(5.30)(5.31)だったと思います．
現在は使っていなくて，実行方法を思い出せたら再現手順書いてみます．

landingTargetは視覚情報を用いて着地高さ・角度を決めるときに使っていて，D論5.4.1節 ノード1の1. の処理です．
サンプルは佐藤君の整備が一通り終わったらつくると思う．
